pino.js

```diff
@@ -245,12 +252,16 @@ Pino.prototype.asJson = function asJson (obj, msg, num) {
   return data + this.chindings + this.end
 }
 
+function getNoTime () {
+  return ''
+}
+
 function getTime () {
-  return Date.now()
+  return flatstr(',"time":' + Date.now())
 }
 
 function getSlowTime () {
-  return '"' + (new Date()).toISOString() + '"'
+  return flatstr(',"time":"' + (new Date()).toISOString() + '"')
```

Comment:
The time string was being flattened in `lscache` so I opted to do here as well. My benchmarks aren't showing any real hit, but I can remove it.